frontend/apps/web/components/providers/unify-provider.tsx

```diff
@@ -49,20 +49,6 @@ export function UnifyIdentifier(): ReactElement {
     ) {
       return;
     }
-    // we only want to set the user id if auth is enabled, otherwise it is always the same
-    // so it makes it harder to identify unique posthog sessions when running in un-auth mode.
-    const userId = systemAppConfig?.isAuthEnabled
```

Comment:
I think this entire UnifyIdentifier should be removed, it's not actually doing anything.